src/MessageBird/Client.php

```diff
@@ -2,179 +2,129 @@
-    const VOICEAPI_ENDPOINT = 'https://voice.messagebird.com';
-    const PARTNER_ACCOUNT_ENDPOINT = 'https://partner-accounts.messagebird.com';
-    const NUMBERSAPI_ENDPOINT = 'https://numbers.messagebird.com/v1';
-
-    const ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX = 'ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX';
-    const CONVERSATIONSAPI_WHATSAPP_SANDBOX_ENDPOINT = 'https://whatsapp-sandbox.messagebird.com/v1';
+    public const ENDPOINT = 'https://rest.messagebird.com';
+    public const CONVERSATIONSAPI_ENDPOINT = 'https://conversations.messagebird.com/v1';
+    public const VOICEAPI_ENDPOINT = 'https://voice.messagebird.com';
+    public const PARTNER_ACCOUNT_ENDPOINT = 'https://partner-accounts.messagebird.com';
+    public const NUMBERSAPI_ENDPOINT = 'https://numbers.messagebird.com/v1';
 
-    const CLIENT_VERSION = '2.0.0-prerelease';
-
-    /**
-     * @var string
-     */
-    protected $endpoint = self::ENDPOINT;
+    public const ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX = 'ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX';
+    public const CONVERSATIONSAPI_WHATSAPP_SANDBOX_ENDPOINT = 'https://whatsapp-sandbox.messagebird.com/v1';
 
+    public const CLIENT_VERSION = '2.0.0-prerelease';
     /**
      * @var Resources\Messages
      */
     public $messages;
-
     /**
      * @var Resources\Contacts
      */
     public $contacts;
-
     /**
      * @var Resources\Groups
      */
     public $groups;
-
     /**
      * @var Resources\EmailMessage
      */
     public $emailmessages;
-
     /**
      * @var Resources\VoiceMessage
      */
     public $voicemessages;
-
     /**
      * @var Resources\Hlr
      */
     public $hlr;
-
     /**
      * @var Resources\Verify
      */
     public $verify;
-
     /**
      * @var Resources\Balance
      */
     public $balance;
-
     /**
      * @var Resources\Lookup
      */
     public $lookup;
-
     /**
      * @var Resources\LookupHlr
      */
     public $lookupHlr;
-
     /**
      * @var Resources\MmsMessages
      */
     public $mmsMessages;
-
-    /**
-     * @var Resources\Chat\Message
-     */
-    public $chatMessages;
-
-    /**
-     * @var Resources\Chat\Channel
-     */
-    public $chatChannels;
-
-    /**
-     * @var Resources\Chat\Platform
-     */
-    public $chatPlatforms;
-
-    /**
-     * @var Resources\Chat\Contact
-     */
-    public $chatContacts;
-
     /**
      * @var Resources\Voice\Calls
      */
     public $voiceCalls;
-
     /**
      * @var Resources\Voice\CallFlows
      */
     public $voiceCallFlows;
-
     /**
      * @var Resources\Voice\Legs
      */
     public $voiceLegs;
-
     /**
      * @var Resources\Voice\Recordings
      */
     public $voiceRecordings;
-
     /**
      * @var Resources\Voice\Transcriptions
      */
     public $voiceTranscriptions;
-
     /**
      * @var Resources\PhoneNumbers
      */
     public $phoneNumbers;
-
     /**
      * @var Resources\AvailablePhoneNumbers
      */
     public $availablePhoneNumbers;
-
```

Comment:
Why remove these lines?

Reply:
It was automatic cleanup on PSR-12 standards.